lib/web/git_servers.go

```diff
@@ -93,5 +93,5 @@ func (h *Handler) gitServerDelete(_ http.ResponseWriter, r *http.Request, p http
 		return nil, trace.Wrap(err)
 	}
 
-	return nil, clt.GitServerClient().DeleteGitServer(r.Context(), name)
+	return OK(), clt.GitServerClient().DeleteGitServer(r.Context(), name)
```

Comment:
this isn't being used yet so I just changed it.

should i just go ahead and convert this [files](https://github.com/gravitational/teleport/blob/master/lib/web/userevent.go) success response to `OK()`?

i don't see why not...